bin/kondo.sh

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+# Convenience for running clj-kondo against all the appropriate directories.
+
+find modules/drivers shared enterprise/backend \
```

Comment:
Doesn't this miss `.`?

Reply:
that's intentional so we don't search a million `.git` or `node_modules` directories. `src` and `test` are included explicitly in the last line of the script